### ISSUE TITLE
aria-description not valid on buttons and inputs

### ISSUE THREAD

**Issue by TheGeeps** [validator/validator#1155]
### Minimum not working example

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>A Title</title>
  </head>
  <body>
	<form>
		<button title="Sort by Date" aria-label="Sort by Date" aria-description="Currently sorted by subject" type="submit">
			Date
		</button>
	</form>
  </body>
</html>
```

### Validation error

```
Error: Attribute aria-description not allowed on element button at this point.
```

### Additional information

In my current project I've made extensive use of the new aria-desciption attribute to provide additional information to visually impaired users. This information does not exist anywhere else in the DOM and is obvious to sighted users from context. The validator does neither accept aria-description as an attribute for buttons nor for inputs. [According to the specification](https://w3c.github.io/aria/#aria-description), the attribute should be valid on all elements. This is a blocker for me as the validator is part of our acceptance criteria.

Please consider allowing aria-description on all markup elements.

**Comment by sideshowbarker** [comment]
I think the checker doesn’t yet support `aria-description` at all yet — regardless of the element or role

**Comment by TheGeeps** [comment]
I've tried a few different elements and you are correct. It's not yet supported on any element.

Is this something that might be added in the near future?

**Comment by sideshowbarker** [comment]
> Is this something that might be added in the near future?

Yes — I have a list of about 30 open issues for bringing the checker in line with the current conformance requirements in the ARIA in HTML and ARIA specs. I am aiming to the necessary changes for all those issues (including this one) added to the checker within the next two weeks.

**Comment by TheGeeps** [comment]
I really appreciate that.

Thank you for the quick replies and help.
